auth-web/src/components/auth/AffiliatedEntityList.vue

```diff
@@ -143,7 +143,7 @@ export default class AffiliatedEntityList extends Vue {
       redirectURL = ConfigHelper.getNewBusinessURL() + 'create?nrNumber=' + business.businessIdentifier
     } else {
       ConfigHelper.addToSession(SessionStorageKeys.BusinessIdentifierKey, business.businessIdentifier)
-      redirectURL = ConfigHelper.getCoopsURL() + 'dashboard'
+      redirectURL = `${ConfigHelper.getCoopsURL()}dashboard/${business.businessIdentifier}`
```

Comment:
It's the other way around :)
https://dev.bcregistry.ca/cooperatives/CP0000841
(The trailing slash is optional and I made the dashboard the root (/) page.)

Reply:
This is now fixed.